src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogComputerPublisher.java

```diff
@@ -80,8 +79,14 @@ protected void doRun() throws Exception {
             // Add JenkinsUrl Tag
             globalTags = TagsUtil.addTagToTags(globalTags, "jenkins_url", DatadogUtilities.getJenkinsUrl());
             for (Computer computer : computers) {
+                Map<String, Set<String>> tags = TagsUtil.merge(
+                        DatadogUtilities.getComputerTags(computer), globalTags);
                 nodeCount++;
+                int isOnline = 1;
```

Comment:
You can have a single boolean for that instead of two integers

Reply:
I couldn't find a way to do it cleanly 😢

Reply:
yeah it's good enough like this 👍